pandas/core/window/ewm.py

```diff
@@ -52,9 +58,38 @@ class EWM(_Rolling):
         Divide by decaying adjustment factor in beginning periods to account
         for imbalance in relative weightings
         (viewing EWMA as a moving average).
+        When ``adjust=True`` (default), the EW function is calculated using
```

Comment:
I'd leave a blank line before, so it's more readable in the markdown version. And make the two `When adjust=..` a bullet point, so it's easier to read in the html.

Reply:
Introducing bullet-points produces these warnings:
![image](https://user-images.githubusercontent.com/12001304/75982279-8b40f300-5ee6-11ea-9d22-9da33edd534f.png)

And the html file: 
![image](https://user-images.githubusercontent.com/12001304/75982301-9ac03c00-5ee6-11ea-9f55-4ebfc923d219.png)

So, as I understand, if I want everything to look neat I should not break the line. However, if I do that there will be a conflict with the requirement that the line should be <= 88 chars.

Reply:
Not sure I understand the problem. I think should be as easy as using something like:
```
* When adjust=whatever
  continuation lines are indented
  if one is two long to be more than 88 just add
  a line break before it is.

And leave a blank line after the bullet point.
```

Is there any problem with this?

Reply:
Yes, this works, sorry. I didn't indent properly the lines.